postcss.config.js

```diff
@@ -1,7 +1,18 @@
 const autoprefixer = require('autoprefixer');
+const globalManifest = require('./src/Blocks/manifest.json');
+const transformRemsPlugin = require('./postcss/es-transforms-rems');
```

Comment:
please update this to use the helper from frontend-libs

https://github.com/infinum/eightshift-frontend-libs/tree/develop/scripts/postcss